docs/documentation/userguide/installation.md

```diff
@@ -136,13 +136,58 @@ Depending whether HDF5 was installed using *configure* or *CMake*, different set
 
         export HDF5_DIR = /opt/hdf5/1.X.X/shared/cmake/XXX
 
-If your CMake version is above 3.9.X, CMake uses a new findPackage routine, requiring that **HDF5_ROOT** is set
+If your CMake version is above 3.9.X, CMake uses a new `findPackage` routine, requiring that `HDF5_ROOT` is set
 
     export HDF5_ROOT=/opt/hdf5/1.10.0-patch1/mpi/
```

Comment:
sure you want to keep the hdf5 version and mpi? better use the same generic path example as in line 137?

Reply:
resolved in dabb1c8